src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionOrUserDiagnosticTest.cs

```diff
@@ -894,5 +894,91 @@ protected static ImmutableArray<CodeAction> GetNestedActions(ImmutableArray<Code
 
             await TestActionCountAsync(input, outputs.Length, parameters);
         }
+
+        protected static void GetDocumentAndSelectSpanOrAnnotatedSpan(
```

Comment:
Existing code moved up the hierarchy from code fix specific test helper `AbstractUserDiagnosticTest` class to common test helper `AbstractCodeActionOrUserDiagnosticTest`